.github/workflows/checks.yml

```diff
@@ -76,7 +76,7 @@ jobs:
           version: "23.2"
           repo-token: ${{ secrets.GITHUB_TOKEN }}
       - name: Toolchain
-        uses: dtolnay/rust-toolchain@nightly
+        uses: dtolnay/rust-toolchain@nightly-2024-06-13 # 1.79.0 nightly
```

Comment:
This maps our nightly to the same stable version we use for our release builds. This is because using `@nightly` updated to 1.83 which changes in the format rules resulting in lint failing.

Reply:
lint should be happening on nightly no?

Reply:
a138859f6996ba6de5f1a23bbaa8d26fea3cd88d fixed with this commit